pull_request_template.md

```diff
@@ -1,5 +1,29 @@
-Please ensure the following when opening a PR:
-- [ ] Added or updated tests
-- [ ] Updated documentation
-- [ ] Applied style guidelines
-- [ ] Considered generalizability beyond our own use case
+### Purpose
+_Please link to the issue being fixed by this PR:_
+
+- No issue is linked to this PR.
+---
+### Changes
+_Please list out what major changes were made in this PR to address the issue:_
+
+- No changes.
+---
+### Review Instructions
+_Please provide instructions about how should a reviewer test/verify the changes in this PR:_
+
+- No instructions.
+
+---
+### PR Checklist
+_Please ensure the following when opening a PR:_
+
+- [ ] This PR added or updated tests.
+- [ ] This PR updated docstrings or documentation.
+- [ ] This PR applied (WDL/Python) style guidelines.
```

Comment:
It would be useful to have a link to our WDL style guide here

Reply:
Per our offline discussion, I created a follow up ticket for this: https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/secondary-analysis/456